tests/e2e/specs/analysis.cy.js

```diff
@@ -264,4 +264,151 @@ describe('Analysis view', () => {
         })
     })
   })
+
+  describe('Time series visualisation', () => {
+    beforeEach(() => {
+      cy.get('.c-analysis [data-cy=time-series-toggle]')
+        .click()
+        .get('.vue-apexcharts')
+        .should('be.visible')
+      // There should be three tasks in the drop down list when loaded
+      cy
+        .get('.d-flex > .v-autocomplete')
+        .click()
+        .get('.v-list-item')
+        .its('length')
+        .should('eq', 3, { timeout: 10000 })
```

Comment:
Maybe try closing this dropdown and checking it is no longer shown. That is what fixed the `ResizeObserver` error last time: https://github.com/cylc/cylc-ui/pull/1529#discussion_r1372021452